dev-client/src/screens/SoilScreen/ColorScreen/components/SwitchWorkflowButton.tsx

```diff
@@ -82,6 +82,7 @@ export const SwitchWorkflowButton = ({
       title={t('soil.color.confirm_delete.title')}
       body={t('soil.color.confirm_delete.body')}
       actionName={t('soil.color.confirm_delete.action_name')}
+      isConfirmError
```

Comment:
question: to confirm my understanding of react syntax here -- will this default `isConfirmError` to true? or is it being passed from somewhere that i've missed?

Reply:
I believe this is equivalent to `isConfirmError={isConfirmError}`.

Reply:
this making me think i should maybe not use this syntax. it's equivalent to `isConfirmError={true}`. in an object, e.g.
```
{ 
  actionName: t('...'),
  isConfirmError
}
```
is the same as `isConfirmError: isConfirmError`, but in JSX syntax it's different. presumably because JSX tries to look like html, where e.g. `<input type="checkbox" checked/>` means "this is checked"